compute/manifest.yml

```diff
@@ -0,0 +1,118 @@
+pg_settings:
+  # Common settings for primaries and secondaries of all versions.
```

Comment:
Is secondary referring to read replicas/hot standbys? If so, I would prefer that terminology.

Reply:
Looks like you use the term `replicas` elsewhere